environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/request/EnvironmentRequest.java

```diff
@@ -301,6 +304,22 @@ public void setEnvironmentServiceVersion(String environmentServiceVersion) {
         this.environmentServiceVersion = environmentServiceVersion;
     }
 
+    public GcpEnvironmentParameters getGcp() {
+        return gcp;
+    }
+
+    public void setGcp(GcpEnvironmentParameters gcp) {
+        this.gcp = gcp;
+    }
+
+    public CcmV2TlsType getCcmV2TlsType() {
+        return ccmV2TlsType;
+    }
+
+    public void setCcmV2TlsType(CcmV2TlsType ccmV2TlsType) {
```

Comment:
should we just use tlstype? and in case of ccmv1 we just using two_way?

Reply:
only ccmv2, moreover it is applied only for the jumpgate version...